tests/data/small_sky_order1/provenance_info.json

```diff
@@ -44,9 +44,11 @@
                 "chunksize": 500000,
                 "header": "infer",
                 "schema_file": null,
-                "separator": ",",
                 "column_names": null,
-                "type_map": {}
+                "type_map": null,
```

Comment:
Just verifying that the change from empty to `null` is intentional.

Reply:
Meh. This whole file will be going away sometime this week anyway. =D